src/hw-3/models/user.ts

```diff
@@ -0,0 +1,27 @@
+export function UserModel(sequelize: any, DataTypes: any) {
```

Comment:
I think it doesn't have much sense to create userModel each time you use it, better to make it singleton.

Reply:
Fixed.